.goreleaser.yaml

```diff
@@ -22,4 +22,26 @@ snapshot:
 
 changelog:
   use: github-native
+
+brews:
+  - name: cfparams
+    tap:
+      owner: cultureamp
+      name: homebrew-tap
+    commit_author:
+      name: cultureamp-ci
+      email: 36431315+cultureamp-ci@users.noreply.github.com
+
+    folder: Formula
+    homepage: https://github.com/cultureamp/cfparams
+    description: Wrangle parameters for AWS CloudFormation
+
+    test: |
+      system "#{bin}/cfparams --help"
```

Comment:
Will this command succeed?

Reply:
It doesn't, lol

Reply:
Updated the test. Still figuring out the install side of things. Got a feeling it has something to do with the binary name.

Otherwise, I am opting to archive it using `tar` if that were not to scuff anything else.

Reply:
All gee, it was renaming that `install` needed.

Reply:
The test command has been extended, though it still doesn't work. Sussing it out now.

Reply:
This has been fixed! 🙌